docs/source/tutorial/type_check.rst

```diff
@@ -67,7 +67,7 @@ Otherwise this code throws an exception, and the user gets a message like this:
 
    Traceback (most recent call last):
    ...
-   InvalidType: Expect: in_types[0].ndim == 2
+   chainer.utils.type_check.InvalidType: Expect: in_types[0].ndim == 2
```

Comment:
How about remove this line?

Reply:
I could not do it. Do you know how to skip the check?

```
Expected:
    Traceback (most recent call last):
    ...
    Actual: 3 != 2
Got:
    Traceback (most recent call last):
      File "/usr/lib/python3.5/doctest.py", line 1321, in __run
        compileflags, 1), test.globs)
      File "<doctest default[0]>", line 1, in <module>
        utils.type_check.expect(x_type.ndim == 2)
      File "/home/user/.local/lib/python3.5/site-packages/chainer-4.0.0a1-py3.5.egg/chainer/utils/type_check.py", line 514, in expect
        expr.expect()
      File "/home/user/.local/lib/python3.5/site-packages/chainer-4.0.0a1-py3.5.egg/chainer/utils/type_check.py", line 472, in expect
        '{0} {1} {2}'.format(left, self.inv, right))
    chainer.utils.type_check.InvalidType: Expect: in_types[0].ndim == 2
    Actual: 3 != 2
```

Reply:
I don't know it. Sometimes, `...` causes mysterious behavior.

Reply:
How about using indent for `...`?
https://docs.python.org/3.6/library/doctest.html#module-doctest

Reply:
Not to work:

```
Expected:
    Traceback (most recent call last):
        ...
    Actual: 3 != 2
Got:
    Traceback (most recent call last):
      File "/usr/lib/python3.5/doctest.py", line 1321, in __run
        compileflags, 1), test.globs)
      File "<doctest default[0]>", line 1, in <module>
        utils.type_check.expect(x_type.ndim == 2)
      File "/home/user/.local/lib/python3.5/site-packages/chainer-4.0.0a1-py3.5.egg/chainer/utils/type_check.py", line 514, in expect
        expr.expect()
      File "/home/user/.local/lib/python3.5/site-packages/chainer-4.0.0a1-py3.5.egg/chainer/utils/type_check.py", line 472, in expect
        '{0} {1} {2}'.format(left, self.inv, right))
    chainer.utils.type_check.InvalidType: Expect: in_types[0].ndim == 2
    Actual: 3 != 2
```

Reply:
We can use `# doctest: +ELLIPSIS`.
https://docs.python.org/3.6/library/doctest.html#directives

Reply:
I've tried `+ELLIPSIS`, but it also did not work 🤔 

```
.. testoutput::
   :options: -IGNORE_EXCEPTION_DETAIL +NORMALIZE_WHITESPACE +ELLIPSIS

   Traceback (most recent call last):
   ...
   Actual: 3 != 2
```

Reply:
I rather needed to fix others, too although they did not tell me failure 🤔